packages/wonder-blocks-link/src/components/__tests__/link.test.tsx

```diff
@@ -370,7 +370,18 @@ describe("Link", () => {
             expect(icon).toBeInTheDocument();
         });
 
-        test("does not render external icon when there is no target", () => {
+        test("does not render external icon when there is no target and link is external", () => {
```

Comment:
**question:** Is there a test for when `target="_blank" is set and link is relative?